sphinx/texinputs/sphinxlatexliterals.sty

```diff
@@ -765,7 +765,8 @@
       % break at . , ; ? ! /
       \sphinxbreaksviaactive
       % break also at \
-      \let\sphinx@textbackslash\textbackslash
+      \setbox8=\hbox{\textbackslash}%
+      \def\sphinx@textbackslash{\copy8}%
```

Comment:
I should perhaps use a proper scratch box but the contents of `\sphinxupquote` should be a text string hence it seems legit to use say `\box8` as here (as this is enclosed in braces).

The `\textbackslash` is a complex command which checks the current font encoding and triggers a fall-back font encoding (OMS, math character) for example if character not available. So the former method with `\let` did not have expected effect and the `\textbackslash` was trying to redefine itself in a  `\discretionary` context where only boxes are allowed.

Reply:
(I mentioned OMS from my pure latex testing, but in Sphinx context this will be T1 encoding)